packages/apps/transfer/.eslintrc.js

```diff
@@ -1,8 +1,32 @@
+const fs = require('fs');
+
+const prettierOptions = JSON.parse(fs.readFileSync('./.prettierrc', 'utf8'));
 require('@rushstack/eslint-config/patch/modern-module-resolution');
 
 module.exports = {
-  plugins: ['jest'],
+  plugins: ['prettier', 'jest'],
   extends: ['next/core-web-vitals', '@kadena-dev/eslint-config/profile/react'],
   parserOptions: { tsconfigRootDir: __dirname },
   ignorePatterns: ['**/*.spec.ts', '**/*.spec.tsx', '**/generated/**'],
+  rules: {
+    'prettier/prettier': ['error', prettierOptions],
+    '@typescript-eslint/strict-boolean-expressions': 0,
+    'arrow-body-style': [2, 'as-needed'],
+    'comma-dangle': [2, 'always-multiline'],
+    'prefer-template': 2,
+    'import/newline-after-import': 1,
+    'import/no-unresolved': 2,
+    'jsx-a11y/aria-props': 2,
+    'jsx-a11y/role-has-required-aria-props': 2,
+    'jsx-a11y/heading-has-content': 1,
+    'jsx-a11y/mouse-events-have-key-events': 2,
```

Comment:
to improve accessibility for for users without a mouse or touchpad this is also really useful, it forces you to have the same value for onFocus and onBlur, in case you used onMouseOver and onMouseOut